### PR TITLE
Ticket2 swagger and endpoint for file

### PR DESCRIPTION
**Pull request by OlenkaN**

### CODE REVIEW

File: src/main/java/com/example/PicturesToNumber/Controllers/PictureController.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import javax.servlet.http.HttpServletRequest;
+import java.sql.SQLException;
+
+@RestController
+@RequestMapping("/api")
+public class PictureController {
+
+   @RequestMapping(value = "/fileUpload", method = RequestMethod.POST)
+    public @ResponseBody String uploadFile(@RequestPart("file") MultipartFile file, HttpServletRequest req)
+            throws SQLException {
```
Comment (AlexUA89) [inline src/main/java/com/example/PicturesToNumber/Controllers/PictureController.java]: Why do you need SqlException?

File: pom.xml [hunk 2]
```diff
@@ -28,6 +28,27 @@
 			<scope>runtime</scope>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
```
Comment (AlexUA89) [inline pom.xml]: If you use spring-boot-stater-web, you don't need spring-boot-started in POM. Please check what is inside spring-boot-starter-web

File: src/main/java/com/example/PicturesToNumber/Controllers/PictureController.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package com.example.PicturesToNumber.Controllers;
```
Comment (AlexUA89) [inline src/main/java/com/example/PicturesToNumber/Controllers/PictureController.java]: packages should be named with low case symbols

File: src/main/java/com/example/PicturesToNumber/Controllers/PictureController.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.servlet.http.HttpServletRequest;
+import java.sql.SQLException;
+
+@RestController
+@RequestMapping("/api")
+public class PictureController {
+
+   @RequestMapping(value = "/fileUpload", method = RequestMethod.POST)
```
Comment (AlexUA89) [inline src/main/java/com/example/PicturesToNumber/Controllers/PictureController.java]: Don't see java docs

File: src/main/java/com/example/PicturesToNumber/Controllers/PictureController.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+import io.swagger.annotations.ApiParam;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.servlet.http.HttpServletRequest;
+import java.sql.SQLException;
+
+@RestController
```
Comment (AlexUA89) [inline src/main/java/com/example/PicturesToNumber/Controllers/PictureController.java]: Don't see javadocs

File: src/main/java/com/example/PicturesToNumber/Controllers/PictureController.java [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    public @ResponseBody String uploadFile(@RequestPart("file") MultipartFile file, HttpServletRequest req)
+            throws SQLException {
+
+        // using file.
+       if(file.equals(null))
+           return "FILE EMPTY";
+
+        return "success";
+    }
+    @GetMapping("/")
```
Comment (AlexUA89) [inline src/main/java/com/example/PicturesToNumber/Controllers/PictureController.java]: Don't see javadocs or just remove this test method

File: src/main/java/com/example/PicturesToNumber/PicturesToNumberApplication.java [hunk 7]
```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
 
 @SpringBootApplication
```
Comment (AlexUA89) [inline src/main/java/com/example/PicturesToNumber/PicturesToNumberApplication.java:10]: Javadocs?

File: src/main/java/com/example/PicturesToNumber/Controllers/PictureController.java (hunk 1 above)
Comment (OlenkaN) [reply src/main/java/com/example/PicturesToNumber/Controllers/PictureController.java]: Sorry, we really don't need it, I've already changed it

File: pom.xml (hunk 2 above)
Comment (OlenkaN) [reply pom.xml]: Got it, I have removed spring-boot-starter

**Pull request merged**